src/Oscoin/P2P/Connection.hs

```diff
@@ -1,100 +0,0 @@
-{-# LANGUAGE TupleSections #-}
-
-module Oscoin.P2P.Connection
-    ( Connection
-    , connNodeId
-    , connAddr
-    , connSendWire
-    , connRecvWire
-    , connClose
-    , mkConnection
```

Comment:
I wish you did send this PR 1 week ago -- it would have avoided me one less painful module to refactor 😁 

Otoh though, this was the kickstarter of our discussion on the `DigitalSignature` stuff, so perhaps things had to go this way 😉

Reply:
Apologies for that. I did in fact only realise after our discussion that this module is no longer used, and that we can get rid of the orphan `instance ByteArrayAccess LByteString` by simply using strict `ByteString`s for the `Framed` transport.